file-browser/src/model/dataSchema.js

```diff
@@ -4,6 +4,7 @@ declare type AccountT = {
     $key: string, // locally generated uuid, used internally in our app
     client_id: string, // OAuth client_id
     access_token: string, // OAuth token
+    backend: string,
```

Comment:
remove this. This belongs to a different PR. 

Also it shouldn't be `backend`, it should be `service`?